src/components/profile-youtube-playlist/profile-youtube-playlist.js

```diff
@@ -0,0 +1,91 @@
+import { Helmet } from 'react-helmet';
+import React, { useState, useEffect } from 'react';
+import Carousel from '@brainhubeu/react-carousel';
+import '@brainhubeu/react-carousel/lib/style.css';
+import PropTypes from 'prop-types';
+
+const YoutubePlaylist = ({ youtubePlayListId }) => {
+  const [error, setError] = useState(null);
+  const [isLoaded, setIsLoaded] = useState(false);
+  const [items, setItems] = useState([]);
+
+  useEffect(() => {
+    const youtubeApiKey = process.env.YOUTUBE_API_KEY;
+
+    if (youtubePlayListId == '') {
+      return null;
+    }
+
+    fetch(
+      'https://www.googleapis.com/youtube/v3/playlistItems?part=contentDetails&maxResults=50&playlistId=' +
```

Comment:
should we use interpolated strings for readability?

Reply:
Done - used interpolated strings